django_filters/filters.py

```diff
@@ -656,7 +656,7 @@ def normalize_fields(cls, fields):
 
     def build_choices(self, fields, labels):
         ascending = [
-            (param, labels.get(field, pretty_name(param)))
+            (param, labels.get(field, _(pretty_name(param))))
             for field, param in fields.items()
         ]
         descending = [
```

Comment:
Use:
```
        descending = [
            ('-%s' % pair[0], labels.get('-%s' % pair[0], self.descending_fmt % pair[1]))
            for pair in ascending
        ]
```
to support translation of label for descending options (eg, '-username'). There is no reason to not support them, but in some cases can improve UX experience by naming orders in more natural ways.